### PR TITLE
Consistently name "Warehouse Client ID"

### PR DESCRIPTION
**Pull request by gigxz**
Non-HUD:
- [x] Update CSV exports
- [x] Update report views - did some, not all

HUD
- [x] APR
- [x] CAPER
- [x] HUD DQ
- [x] CE APR
- [x] PIT
- [x] HIC - N/A? there is a template `drivers/hud_hic/app/views/hud_hic/shared/_cell_details.haml` but afaict it is not used..
- [x] PATH -- added `destination_client_id` to `HudPathReport::Fy2020::PathClient`
- [x] SPM -- N/A, this one stores the destination client id in the `client_id` field, unlike all the rest which store source client ID there

### CODE REVIEW

**Comment by gigxz** [comment]
@eanders we don't already store destination_client_id for the SPM or PATH. Is it worth adding a column for it to each of those tables?

**Comment by eanders** [comment]
@gigxz , yes, we should add those.  They should be pretty easy to add in the `add_clients` method.

File: drivers/ce_performance/app/models/ce_performance/report.rb [hunk 1]
```diff
@@ -486,7 +486,7 @@ def detail_headers(key: nil) # rubocop:disable Lint/UnusedMethodArgument
       @detail_headers ||= {}.tap do |headers|
         headers.merge!(
           {
-            'client_id' => 'Warehouse Client ID',
+            'client_id' => 'Warehouse Source Client ID',
```
Comment (gigxz) [inline drivers/ce_performance/app/models/ce_performance/report.rb:491]: Do we want to update this report to include the destination ID? Would require a migration

**Comment by gigxz** [comment]
This is ready for review BTW @eanders , all HUD reports are showing destination ID

File: drivers/ce_performance/app/models/ce_performance/report.rb (hunk 1 above)
Comment (eanders) [reply drivers/ce_performance/app/models/ce_performance/report.rb:491]: Yes, we should do this like you did for the PATH report.

**Review by eanders — APPROVED** [review]
Looks great! Thanks!  Just the one report you called out, and it'll be good to merge.

**Comment by gigxz** [comment]
Alright, updated with https://github.com/greenriver/hmis-warehouse/pull/2583/commits/701f63a375b1b77da67cc98fb8c0d79f9da842f7!

**Review by eanders — APPROVED** [review]
Thanks!  Looks great.

**Pull request merged**